### ISSUE TITLE
windows port

### ISSUE THREAD

**Issue by lm8** [MedallionMediaGroup/EPUB3Processor#2]
Doesn't work for Windows as is.  However, I do have some patches so that EPUB3Processor library will build and run on Windows using MinGW.  If you want a copy of the patches, please let me know.

**Comment by casademora** [comment]
Can you submit a pull request?

**Comment by lm8** [comment]
On Fri, December 13, 2013 11:55 am, Saul Mora wrote:

> Can you submit a pull request?

I don't have the code at github at this point.  Is there a good way to
send patches?

**Comment by MedallionMediaGroup** [comment]
Yes, we plan on supporting this library but need help from the open source community.

Master branch has been updated with latest api editions. To summit your Win x86 changes you will have to fork the repo into your own Git repository, add in your changes and submit a pull request.

@Im8
First of all, thank you for porting this in to Win x86. Can you reassure that Mingw is not using any 3rd party dependencies. We are trying to keep this code clean and portable with no dependencies on 3rd party libraries. Can you compile in MS Visual Studio using just standard libraries? Thanks again.

**Comment by lm8** [comment]
On Mon, December 16, 2013 9:45 am, Medallion Media Group wrote:

> Master branch has been updated with latest api editions. To summit your
> Win x86 changes you will have to fork the repo into your own Git
> repository, add in your changes and submit a pull request.

I don't use git for my development and therefore can't send you a pull
request.

> First of all, thank you for porting this in to Win x86. Can you reassure
> that Mingw is not using any 3rd party dependencies. We are trying to keep
> this code clean and portable with no dependencies on 3rd party libraries.
> Can you compile in MS Visual Studio using just standard libraries? Thanks
> again.

I haven't used Visual Studio in years.  I found it too buggy.  I actually
found a serious memory leak with the product years ago and was conversing
with their support people about it.  Haven't used it for development since
the problem.  Given the seriousness of the issue, I personally would not
want to have production quality code compiled with Visual Studio.  Not
sure what you're referring to with regards to 3rd party dependencies. 
There shouldn't be any extra libraries needed to build unless your library
requires them and they're in your makefile.
